flags/14.0/vtbackup.txt

```diff
@@ -1,318 +1,170 @@
-      --master_connect_retry duration                                    Deprecated, use -replication_connect_retry (default 10s)
-      --mem-profile-rate int                                             deprecated: use '-pprof=mem' instead (default 524288)
-      --min_backup_interval duration                                     Only take a new backup if it's been at least this long since the most recent backup.
-      --min_retention_count int                                          Always keep at least this many of the most recent backups in this backup storage location, even if some are older than the min_retention_time. This must be at least 1 since a backup must always exist to allow new backups to be made (default 1)
-      --min_retention_time duration                                      Keep each old backup for at least this long before removing it. Set to 0 to disable pruning of old backups.
-      --mutex-profile-fraction int                                       deprecated: use '-pprof=mutex' instead
-      --mycnf-file string                                                path to my.cnf, if reading all config params from there
-      --mycnf_bin_log_path string                                        mysql binlog path
-      --mycnf_data_dir string                                            data directory for mysql
-      --mycnf_error_log_path string                                      mysql error log path
-      --mycnf_general_log_path string                                    mysql general log path
-      --mycnf_innodb_data_home_dir string                                Innodb data home directory
-      --mycnf_innodb_log_group_home_dir string                           Innodb log group home directory
-      --mycnf_master_info_file string                                    mysql master.info file
-      --mycnf_mysql_port int                                             port mysql is listening on
-      --mycnf_pid_file string                                            mysql pid file
-      --mycnf_relay_log_index_path string                                mysql relay log index path
-      --mycnf_relay_log_info_path string                                 mysql relay log info path
-      --mycnf_relay_log_path string                                      mysql relay log path
-      --mycnf_secure_file_priv string                                    mysql path for loading secure files
-      --mycnf_server_id int                                              mysql server id of the server (if specified, mycnf-file will be ignored)
-      --mycnf_slow_log_path string                                       mysql slow query log path
-      --mycnf_socket_file string                                         mysql socket file
-      --mycnf_tmp_dir string                                             mysql tmp directory
-      --mysql_auth_server_static_file string                             JSON File to read the users/passwords from.
```

Comment:
I'll have to check on this. these flags are intended for people to use to generate a custom my.cnf versus what we generate by default. Perhaps they should stay on vtbackup because it should run with the same my.cnf as vttablet.

Reply:
Yes, we do need to add these back on vtbackup. We might want to deprecate the mycnf template functionality in future.

Reply:
https://github.com/vitessio/vitess/pull/11495/files

Reply:
Sorry @rsajwani  I meant the mycnf_* flags.
mysql_auth flags are not needed.

Reply:
and looks like the mycnf part has already been fixed.